nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -34,6 +34,9 @@ Keep an old app running and swap the ports when the new one is booted.
           host = "*4";
```

Comment:
L30: `lib.types.attrs` is wrong, it's `(pkgs.formats.yaml {}).type` or something like that.

Reply:
yes I got this thanks!